docs/content/Coding Milestones/PoC2023q1/commands.md

```diff
@@ -474,18 +475,21 @@ kubestellar-version
 ## Creating SyncTarget/Location pairs
 
 In this PoC, the interface between infrastructure and workload
-management is inventory API objects.  Specifically, for each edge
-cluster there is a unique pair of SyncTarget and Location objects in a
-so-called inventory management workspace.  These kinds of objects were
+management is inventory API objects.  Specifically, for each workload
+execution cluster there is a unique pair of SyncTarget and Location
+objects in a so-called inventory space.  These kinds of objects were
```

Comment:
The line below (about kcp) is not important, should we remove it ?

Reply:
kcp TMC still exists and some people know that KubeStellar has history with TMC, so I think it is best to leave this disclaimer in for a while. I think that once the message of "kcp decoupling" is well established that should demolish any lingering expectation of TMC usage and then we can remove this disclaimer.